src/Compilers/Core/Portable/Syntax/ChildSyntaxList.cs

```diff
@@ -83,17 +83,46 @@ private static int Occupancy(GreenNode green)
             return green.IsList ? green.SlotCount : 1;
         }
 
+        internal struct SlotData(int slotIndex, int precedingOccupantSlotCount, int positionAtSlotIndex)
```

Comment:
>SlotData(int slotIndex, int precedingOccupantSlotCount, int positionAtSlotIndex) [](http://example.com/codeflow?start=24&length=80)

The compiler code generally prefers explicit constructors to avoid the chance of unnecessary capture.

Reply:
Done!